simtools/model/camera.py

```diff
@@ -809,10 +814,15 @@ def plotPixelLayout(self):
         plt.tick_params(axis="both", which="major", labelsize=15)
 
         self._plotAxesDef(plt, self._pixels["rotateAngle"])
+        description = "For an observer facing the camera"
+        if cameraInSkyCoor and not isTwoMirrorTelescope(self._telescopeModelName):
+            description = "For an observer behind the camera looking through"
+        if isTwoMirrorTelescope(self._telescopeModelName):
+            description = "For an observer looking from secondary to camera"
```

Comment:
I guess it is clear here that it is the secondary?

Reply:
I had `the` in the sentence but it became too long in the plot. I assume it is clear enough for the people looking at the these plots.